lint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Very simple lint driver. This is primarily because shellcheck needs to
+# be told the files to check as arguments.
+
+set -e
+
+PROG=$(basename "$0")
+PROGDIR=$(dirname "$0")
+
+cd "$PROGDIR"
+flake8
+shellcheck \
+    "$PROG" \
+    sync-s3.sh
```

Comment:
GLib lists the files to check [using `git ls-files`](https://gitlab.gnome.org/GNOME/glib/-/blob/main/.gitlab-ci/run-shellcheck.sh#L7), which has worked reliably for us. It does depend on `lint.sh` being run inside a git checkout though — I don’t know if that is the case.

Reply:
Nice, I didn't think of that. I'm pretty sure this is only ever going to run from a checkout.